### PR TITLE
Support `fixed` in SingleExpr

### PR DESCRIPTION
**Pull request by sergey-tihon**
Fix candidate for #2112 
p.s. I have no idea what I am doing, so please review

### CODE REVIEW

**Comment by baronfel** [comment]
>  I have no idea what I am doing, so please review

@sergey-tihon your humility is an inspiration, as always 😂

File: src/Fantomas/CodePrinter.fs [hunk 1]
```diff
@@ -1250,6 +1250,7 @@ and genExpr astContext synExpr ctx =
                 genTriviaFor SynExpr_YieldOrReturnFrom_ReturnBang returnBangKeyword !- "return! "
             | Do doKeyword -> genTriviaFor SynExpr_Do_Do doKeyword !- "do "
             | DoBang doBangKeyword -> genTriviaFor SynExpr_DoBang_DoBang doBangKeyword !- "do! "
+            | Fixed fixedKeyword -> genTriviaFor SynExpr_Fixed fixedKeyword !- "fixed "
```
Comment (sergey-tihon) [inline src/Fantomas/CodePrinter.fs:1253]: This line surprised me a little bit.
You already had `SynExpr_Fixed` but it was not supported in `SingleExp` - strange!

**Review by nojaf — CHANGES_REQUESTED** [review]
Thank you for picking this one up @sergey-tihon!
You are quite close, but there is a bit of a twist with the `fixed` keyword.

In its current form, this following snippet will loose the comment

```fsharp
let a = fixed // comment
                     b
```

with my additional pointers.

Would you mind adding a test for this as well?

**Comment by sergey-tihon** [comment]
@nojaf what is the expected formatting your sample?

```fsharp
let a = fixed // comment
                     b
```

current version produce

```fsharp
let a = // comment
    fixed b
```

and it looks good for me.

File: src/Fantomas.Tests/SingleExprTests.fs [hunk 2]
```diff
@@ -213,3 +213,13 @@ let ``comment after address of tokens`` () =
 && // comment
     foobar
 """
+
+[<Test>]
+let ``parse fixed in SingleExpr`` () =
```
[the remark this answers is not in the document]
Comment (sergey-tihon) [reply src/Fantomas.Tests/SingleExprTests.fs]: Done, thx

**Comment by nojaf** [comment]
The comment should stay after the `fixed` keyword, similar to how [upcast](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAbRAXMBDMAvGAK4AO0uAzjgPQ1hwC2jiAdjgIwA6ATl6xEFDhEAEa9%2BE1nTCYc0QmABmASwAeieGBlMW7MACYpMkacHxIAGnBRK2LCtYBzSqjABtGxFAjIAEiwAT1JkNEhHLCsvQX8AN1x0YlCPaKEABmszSABJViR2AGUVAC9kTNMAFlSwAF1ogF9y72r-IJC3cPYo3z94xOTPMw4m4UgAWVw1ABlHRCm2JywAC27TDgM06rqhRuifUYDg5Mg2eAB5JRnWRAKgzFWhOISkt0HTAxHHgFE8i6uy6oQSDoJSQLYNEb7R6HdphUSwWDoB4xXrPAaAsAAZk%2BMVydh4WAAYo4EgA5RAAd3Qs2RQiwPCS4J2kJaMOO8MRtKgqP6rwxFRxQIKiEYKjg6FgrAAglgfudLjTBcoEnYmYJdkIoSi2uyEUicU9eWg3iIAKxKyAFUi4aCIABCiCUsB4iAACrgeLgWFhEDwuRB6YyRNt1SyejqOhz9ay%2Bi9jRiAGwWq02%2B2O51zWAU30UOy5WKwCgOSX%2BsCBxBqiAawRaoFsyN6rmGuMpMwAdmT1ttDqdLoAqqQQjxc4h84XcMXWKWlCqK8GIXtWRG4Y2DTyWybhAAOTupnsZgDC6ColAPkuoDOgWGd09nlbA1ea4aODc5a9j6LMAE5d930y6xhFURfVvdBVXnZlF2fWEoCjJt10-NYMgxS0uzTXtEDPCUpyVGcwLnYQQyrMMDmXWDVxjNE%2BSGYYUJTW0pSUH0eDPZhcFLct70fMBa25MjIDg98qPjIYPjotD9xdYVRXFEtcLvCDQyg0iXxXN9KKNVs1mxcTU0Y5jpLFRE5IxTjFOI5ToX4wSNI3DEOAFFDcnyLAzlYAAVHhAgAdRUZZQPAwiF01JdVPI9Seg-ai1nNXSGJ4WBiDyAARRBqVFZiOIZAihCIh8SKssLOkiITNM3IQOCTFCJjUbIlHcpY2C%2BfCCiWZ0sF8%2BB-KVCpNnM-LLO1IqIngqKRLWDtqsmXJVDUM4hwnZ0vjUUgXUoSgVBMsxTT6oLIJC6DjhG0q7KGHcprUAAlRA4B4eBOu6-ldty4Ka1CmDitG4StJEDgfwu67bvgUliEYYCeAubIfUYVwlQ4LiCqGj7GGIdAHGpa5CWdRgJ2Y9yXxOxCRA2C1AedeAxlR9HZixngcawTKLWgJYPRtZiAH0KRULqVgRwa63447bKJ4QDForIaqlHhPUCM4eBmagHpWHrnsEPLuN41phq6QnouJsSJcmKWZblhWsBBsHfUh6HYfsvmDpU5GqZUDHEFp%2Bm8YJ4W9dFnSsmN3BZfllRqEptGXZp7HcZApmWc9K9fU57n-Ptt7Do6IXIu%2B8rBAMRzDbUAA1NE7UceBHCcJXSy3VWIHVxGBe1krvfG4nYoLwkkqvTbWFLvIK6rlXU6fR2jp1luftFqqC%2BS2AsAAcWwZbVsQdae8HjEduHnj3t1CKDjGyehAMSaJed127QlaAAGs7Xj6-sEoNyCi9TDEVBnCMTwwKXv2tPR9fNGLOZUMQGHOlkckVJZgOiwNmNg%2BMQipVUKwPyPcpR5EApbHgtszDfxymrV6I9CofRssA067x-pZAANKIEQKQOqDU2C5Bfiwf4AV8F10ITvdOYRM4H2zhiTEyEC7NVFJQJYG8zC9W3presakgH8JAWYTE4tTCWgrpgKWThQZsA6ogUQYwEAAlwQpPaSkHbEL3go6Eh8c4QExAbNRUpqROFYJ3Vg3dJRFFcROYgLp3KwGcroicPd2EyN3oAr6SjTCYj9k4tGvpWAhNiJmZwmDwZIMcKg88YT%2BoawifIqJ5CRCYnzmosODh-hTC9KIeAuAjywA2s4SBrtclmIshYpGViikiyEJiduajkoh1wKITAoiQ5LAKIEdgkw2m-3Mf-SxkTdat2EJiaeaiaF0KCewXId9cAeOVl-Ux8yOmLK6csiedisSnzUZfA519-iUClolPI5JqCaAqRHa4y81obRyUqMy7SBqdMbiQiiZDemCExOAu5HpJKICGZQaAPAVCiiST6eAfYUGSlStAY8npJxzIIX-IhFzCkrKPtCyhajbioqvIYpAxLOE7BsDUco4RKCEg2m4PBIB6hAA) does it.

Should be 
```fsharp
let a = 
    fixed // comment
        b
```

**Comment by nojaf** [comment]
Also, https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#format-your-changes and https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#run-a-local-build ;)

File: src/Fantomas.Tests/SingleExprTests.fs [hunk 3]
```diff
@@ -11,7 +11,7 @@ let ``comment after do keyword, 1875`` () =
         """
 do  // Comment DO
         let x = 1
-        f (x) 
+        f (x)
```
[the remark this answers is not in the document]
Comment (sergey-tihon) [reply src/Fantomas.Tests/SingleExprTests.fs]: Sorry, all my IDEs remove spaces on save

File: src/Fantomas/CodePrinter.fs (hunk 1 above)
Comment (sergey-tihon) [reply src/Fantomas/CodePrinter.fs]: Thank for so detailed explanation! I hope that I did it right =)

**Comment by sergey-tihon** [comment]
Thank you @nojaf ! It was a pleasure to work on PR with such guidance.

**Pull request merged**